### PR TITLE
bring down push bodies as part of push thread load request

### PR DESCRIPTION
**Pull request by mmaxim**
Should make it so we always attempt to use the message data on the push notificationl

### CODE REVIEW

File: shared/ios/Keybase/AppDelegate.m [hunk 1]
```diff
@@ -163,18 +163,6 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
       NSLog(@"Remote notification handle finished...");
     });
   } else if (type != nil && [type isEqualToString:@"chat.newmessage"]) {
-    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^(void){
```
Comment (joshblum) [inline shared/ios/Keybase/AppDelegate.m]: does android have an equivalent we should remove as well?

File: go/chat/uithreadloader.go [hunk 2]
```diff
@@ -563,15 +573,24 @@ func (t *UIThreadLoader) shouldIgnoreError(err error) bool {
...
-func (t *UIThreadLoader) singleFlightConv(ctx context.Context, convID chat1.ConversationID) (context.Context, context.CancelFunc) {
+func (t *UIThreadLoader) singleFlightConv(ctx context.Context, convID chat1.ConversationID,
+	reason chat1.GetThreadReason) (context.Context, context.CancelFunc) {
 	t.activeConvLoadsMu.Lock()
 	defer t.activeConvLoadsMu.Unlock()
 	convIDStr := convID.ConvIDStr()
-	if cancel, ok := t.activeConvLoads[convIDStr]; ok {
-		cancel()
+	if convLoad, ok := t.activeConvLoads[convIDStr]; ok {
+		if convLoad.reason != chat1.GetThreadReason_PUSH {
```
Comment (joshblum) [inline go/chat/uithreadloader.go]: why bother storing this conv load if we don't want to cancel it?

Comment (joshblum) [reply go/chat/uithreadloader.go]: also should this map get cleaned up after the call?

File: shared/ios/Keybase/AppDelegate.m (hunk 1 above)
Comment (mmaxim) [reply shared/ios/Keybase/AppDelegate.m]: No

File: go/chat/uithreadloader.go (hunk 2 above)
Comment (mmaxim) [reply go/chat/uithreadloader.go]: Sure I guess that could be better. I don't think the cleanup is necessary though

**Pull request merged**